src/components/dialoghelper/dialogHelper.js

```diff
@@ -0,0 +1,486 @@
+define(['appRouter', 'focusManager', 'browser', 'layoutManager', 'inputManager', 'dom', 'css!./dialoghelper.css', 'scrollStyles'], function (appRouter, focusManager, browser, layoutManager, inputManager, dom) {
```

Comment:
not reviewing this file until it's either removed (if it's a duplicate) or explicitly said it's a new file... skipping for now

Reply:
This is strange, it is a rename only from dialoghelper.js to dialogHelper.js. the rename is referenced separately from this file. What i am missing here?

Reply:
I was able to solve this issue.